script/before_install.sh

```diff
@@ -30,14 +31,6 @@ then
     fi
 fi
 
-#-------------------------------------------------------------------------------
-# Boost.
-echo $ALPAKA_CI_BOOST_BRANCH
```

Comment:
The way to split strings in bash without assuming always the correct number of digits is https://stackoverflow.com/a/5257398

Reply:
Do you mean, that is the sense of this chunk of code? An example for spiting strings. If yes, I can bring it back and comment it. I removed it, because it has no usage and confused me.

Reply:
I looked to the wrong side of the diff^^ NO need to reintroduce the version splitting!